src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts

```diff
@@ -0,0 +1,19 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class clearIndexErrorsCommand extends commandBase {
+    constructor(private indexesNames: string[], private db: database) {
+        super();
+    }
+
+    execute(): JQueryPromise<void> {
+        const args =  this.indexesNames ? { name: this.indexesNames } : "";
+        const url = endpoints.databases.index.indexesErrors + this.urlEncodeArgs(args);
+
+        return this.del<void>(url, null, this.db)
+            .done(() => this.reportSuccess(`Successfully cleared errors from ${this.indexesNames ? 'selected' : 'all'} indexes`));
```

Comment:
I wouldn't assume here that we removed selected from this command perspective. If you invoke this method from other place, like edit index view then this text won't be correct.

Reply:
done